improver/cli/__init__.py

```diff
@@ -95,8 +97,8 @@ def add_docstring(self, docstring, *args, **kwargs):
 
 class DocutilizeClizeHelp(ClizeHelp):
     """Subclass to build Napoleon docstring from subject."""
-    def __init__(self, subject, owner,
-                 builder=HelpForNapoleonDocstring.from_subject):
+
+    def __init__(self, subject, owner, builder=HelpForNapoleonDocstring.from_subject):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Useless super delegation in method '__init__'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5385606)